core/src/main/res/values/arrays.xml

```diff
@@ -233,6 +233,16 @@
         <item>3</item>
     </string-array>
 
+    <string-array name="nav_drawer_feed_filter_values">
+        <item>0</item>
+        <item>1</item>
+    </string-array>
+
+    <string-array name="nav_drawer_feed_filter_options">
+        <item>@string/none_label</item>
```

Comment:
Please create a new string for this (`no_filter_label`, "No filter"). There is a difference between filter/none and select/none in German, even if it is both "none" in English. I created #4305 to rename the string, so this is more clear.